lib/rules/web/admin_console/4.3/check_links.xyaml

```diff
@@ -165,3 +165,43 @@ check_customized_oc_download_links:
     text: 下载oc命令行
     link_url: mirror.openshift.com/pub/openshift-v4/clients/oc
   action: check_link_and_text
+
+# --- links on navigator menu ---
+check_Administration_menu_for_normal_user:
```

Comment:
Mix use of both upper and lower cases letters. Is it expected ? @SSshahan  @yapei

Reply:
let me update

Reply:
@liangxia  I have updated  the file and related to feature file. https://github.com/openshift/cucushift/pull/7485 please help to merge